### PR TITLE
docs(example): introduce v18 example

### PR DESCRIPTION
**Pull request by asudoh**
### Related Ticket(s)

Refs #3359.

### Description

Introduces a CodeSandbox example that uses both Carbon for IBM.com Web Components library and legacy IBM.com Northstar library.

### Changelog

**New**

- A CodeSandbox example that uses both Carbon for IBM.com Web Components library and legacy IBM.com Northstar library.

### CODE REVIEW

File: packages/web-components/README.md [hunk 1]
```diff
@@ -114,6 +114,44 @@ Once you do that, you can use our components as easy as using HTML tags, like:
 
 > 💡 Above CodeSandbox example uses [`html-webpack-plugin`](https://webpack.js.org/plugins/html-webpack-plugin/) to let [WebPack server](https://webpack.js.org/configuration/dev-server/) serve the `.html` file, but you can use other means to serve `.html` files, for example, using [Express](http://expressjs.com) server.
 
+### Using with legacy IBM.com Design System (Northstar)
+
+[Shadow DOM](https://w3c.github.io/webcomponents/spec/shadow/), one of the standards Carbon for IBM.com Web Components library is created on top of, isolates IBM.com Web Components library's component style from your application style, that means those two styles won't adversely affect each other.
```
Comment (jeffchew) [inline packages/web-components/README.md:119]:
```suggestion
[Shadow DOM](https://w3c.github.io/webcomponents/spec/shadow/), one of the standards for Carbon for IBM.com Web Components is created on top of, isolates the web component styles from the application styles. This means those two styles won't adversely affect each other.
```

Comment (asudoh) [reply packages/web-components/README.md]: Thank you for the suggestion @jeffchew - Incorporated.

File: packages/web-components/README.md [hunk 2]
```diff
@@ -114,6 +114,44 @@ Once you do that, you can use our components as easy as using HTML tags, like:
...
+
+If your have a website that currently runs on [legacy IBM.com Design System (Northstar)](https://www.ibm.com/standards/web/v18/), such isolation helps you for gradual migration from legacy IBM.com Northstar style to Carbon for IBM.com style. You can simply use both technologies in your page. Here's an example with Carbon for IBM.com masthead and legacy IBM.com Northstar footer:
```
Comment (jeffchew) [inline packages/web-components/README.md:121]:
```suggestion
For applications that are currently running on [legacy IBM.com Design System (Northstar)](https://www.ibm.com/standards/web/v18/), such isolation will assist with gradual migration from legacy IBM.com Northstar styles to Carbon for IBM.com styles. Both technologies can co-exist safely in the same application. Here is an example with the Carbon for IBM.com masthead and legacy IBM.com Northstar footer:
```

Comment (asudoh) [reply packages/web-components/README.md]: Thanks @jeffchew! 🙇

File: packages/web-components/examples/codesandbox/usage/v18/index.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            <li><a href="https://www.ibm.com/privacy/us/en/">Privacy</a></li>
+            <li><a href="https://www.ibm.com/legal/us/en/">Terms of use</a></li>
+            <li>
+              <a href="https://www.ibm.com/accessibility/us/en/"
+                >Accessibility</a
+              >
+            </li>
+          </ul>
+        </div>
+      </footer>
```
Comment (jeffchew) [inline packages/web-components/examples/codesandbox/usage/v18/index.html:93]: Wonder if we should include `dds-footer-container` here instead too?

Comment (asudoh) [reply packages/web-components/examples/codesandbox/usage/v18/index.html:93]: Good question @jeffchew - Wanted to demonstrate major Carbon for IBM.com component and major Northstar component can co-exist. That's why the footer is of Northstar. Please let me know if you have any better idea to demonstrate such co-existence. Thanks!

Comment (jeffchew) [reply packages/web-components/examples/codesandbox/usage/v18/index.html:93]: Ok, since it's an example I think it should be enough!

**Review by jeffchew — APPROVED** [review]
LGTM!

**Pull request merged**